### PR TITLE
style: simplify overview page - make banner fit in the actual header

### PR DESCRIPTION
**Pull request by shawnyama**
# Description

Note the background fills the bottom with white. Before the banner background was used as the main background. The banner background is moved to the header of `tera-asset.vue` when Overview is chosen.

image

### CODE REVIEW

**Comment by mwdchang** [comment]
Not sure if it is possible or how difficult it is, can we leverage pageType/assetType instead of an isOverView predicate? It makes tera-asset a little more idiomatic in terms of reading.

**Comment by YohannParis** [comment]
I don't understand the need to do this task?

**Comment by shawnyama** [comment]
> I don't understand the need to do this task?

Overview page wasn't fitting the space properly. Just got annoyed looking at it while removing tabs.

File: packages/client/hmi-client/src/components/asset/tera-asset.vue [hunk 1]
```diff
@@ -14,7 +14,11 @@
 			</aside>
 		</header>
 		<template v-if="!hideIntro">
-			<header id="asset-top" ref="headerRef">
+			<header
+				id="asset-top"
+				:class="pageType === ProjectPages.OVERVIEW && 'overview-banner'"
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/asset/tera-asset.vue:19]: Use [class binding](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes) instead.
```suggestion
				:class="{ 'overview-banner': pageType === ProjectPages.OVERVIEW }"
```

**Review by YohannParis — APPROVED** [review]

**Pull request merged**